src/github.com/couchbase/sync_gateway/db/changes.go

```diff
@@ -357,3 +439,65 @@ func (context *DatabaseContext) WaitForPendingChanges() (err error) {
 	}
 	return
 }
+
+// Late Sequence Feed
+// Manages the changes feed interaction with a channels cache's set of late-arriving entries
+type lateSequenceFeed struct {
+	active       bool   // Whether the changes feed is still serving the channel this feed is associated with
+	lastSequence uint64 // Last late sequence processed on the feed
```

Comment:
despite being longer and slightly redundant, `lastLateSequence` might be a more descriptive name for this variable.  thoughts?